.gitignore

```diff
@@ -12,3 +12,9 @@ matrix.log
 glitch_plan*.yaml
 .tox
 .cache
+# exclude venv in case of "virtualenv ."
+lib/
+bin/
+local/
+include/
+pip-selfcheck.json
```

Comment:
Fair enough.  I had originally hoped I could just add to that file and py.test would do the right thing, but I had to add the manual `--ignore` args anyway.